packages/flutter/lib/src/material/dialog.dart

```diff
@@ -252,10 +269,14 @@ class AlertDialog extends StatelessWidget {
 
     return new Dialog(
       child: new IntrinsicWidth(
-        child: new Column(
-          mainAxisSize: MainAxisSize.min,
-          crossAxisAlignment: CrossAxisAlignment.stretch,
-          children: children,
+        child: new Semantics(
```

Comment:
seems like something we could push to `Dialog`, since all dialogs need this?

Reply:
I guess AlertDialog is a special case because it has a "title". We should make sure we document in Dialog that you're expected to provide route name semantics.

Reply:
maybe have a String argument to Dialog, say semanticTitle, that is normally null, and if non-null it sets the route name?
Then AlertDialog can do the right thing with just wrapping the title if it isn't given an explicit semanticTitle, and if it has a semanticTitle it can pass it on to Dialog and not worry about wrapping title.